src/components/calculator.js

```diff
@@ -1,46 +1,30 @@
 /* eslint-disable react/button-has-type */
+import PropTypes from 'prop-types';
 import { Component } from 'react';
+import BonusList from './ButtonList';
+import Display from './Display';
 
 // eslint-disable-next-line react/prefer-stateless-function
 export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
   render() {
+    const { display, click } = this.props;
```

Comment:
- [ ] Also if an operator is clicked on twice it throws an invalid number error. Kindly fix it.

<img src='https://user-images.githubusercontent.com/33323077/130068056-95211a14-e43d-4e16-9f71-3573852c2a76.png' height='250px'> |